spec/tests/rules/uniqueItems.json

```diff
@@ -11,6 +11,11 @@
         "data": ["foo", "bar", "baz"],
         "valid": true
       },
+      {
+        "description": "array of unique items with strings that are properties of has are valid",
```

Comment:
Description here is bit confusing, everything else is ok. Thank you.

Reply:
meant to write "hash"